gadfly/gap_fill_generator.py

```diff
@@ -30,15 +30,28 @@ def find_named_entities(self):
         return entities
 
     def summarize_sentences(self):
-        fs = FrequencySummarizer()
-        sents = []
-        for span in self._parsed_text.sents:
-            sent = [self._parsed_text[i] for i in range(span.start, span.end)]
-            tokens = []
-            for token in sent:
-                tokens.append(token.text)
-            sents.append(tokens)
-        sentences = fs.summarize(sents, 5)
+        option = 1
+        if option == 0:
+            selector = FrequencySummarizer()
+            sents = []
+            for span in self._parsed_text.sents:
+                sent = [self._parsed_text[i] for i in range(span.start, span.end)]
+                tokens = []
+                for token in sent:
+                    tokens.append(token.text)
+                sents.append(tokens)        
+        else: # I messed w/ the below to get access to the lemma in my summarizer - DSG
+            selector = TF_IDFSummarizer()
+            sents = []
+            for span in self._parsed_text.sents:
+                sents.append([self._parsed_text[i] for i in range(span.start, span.end)])
+        sentences = selector.summarize(sents, round(len(sents)*.2))
+        # print("There are {} sentences sent *TO* TF_IDFSummarizer.".format(len(sents)))
```

Comment:
Will do. But, question: That line (and others) provides optional metrics on/transparency into the performance of the selection algorithm. Should I instead make a different output.txt type thing so that the code runs as expected but allows for the EDA I'm wanting to build out?